cpp/demo/interpolation/main.cpp

```diff
@@ -0,0 +1,75 @@
+// Copyright (C) 2022 Igor A. Baratta and Massimiliano Leoni
```

Comment:
Demo name isn't very informative. It's not clear what this demo is demonstrating.